.changeset/olive-deers-smash.md

```diff
@@ -0,0 +1,6 @@
+---
+'@roadiehq/backstage-plugin-wiz': minor
```

Comment:
Judging from the PR there doesn't seem to be new funcitionality added so these should be patch changes